python/nav/models/manage.py

```diff
@@ -1318,9 +1323,17 @@ class GwPortPrefix(models.Model):
     """
 
     interface = models.ForeignKey(
-        'Interface', on_delete=models.CASCADE, db_column='interfaceid'
+        'Interface',
+        on_delete=models.CASCADE,
+        db_column='interfaceid',
+        related_name="gw_port_prefixes",
+    )
+    prefix = models.ForeignKey(
+        'Prefix',
+        on_delete=models.CASCADE,
+        db_column='prefixid',
+        related_name="gw_port_prefixes",
```

Comment:
See my previous comment about how `gwport` and `swport` without underscores are pretty universal terms in the codebase.